app/assets/stylesheets/_pickup-history.scss

```diff
@@ -15,7 +15,7 @@
     background-color: $red;
   }
 
-  &.no-donation {
+  &.no_donation {
```

Comment:
@seanpdoyle in either case, we are using a less-preferred class naming convention in one place or the other, so your call (I wouldn't say this is critical)